src/LaboratoryResults-3-0/Test/Receive/2-5a-receive-minimalNHG.xml

```diff
@@ -75,27 +74,26 @@
         <!-- Asserts for checking one Observation to be compliant with the wiki page on
         https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->     
         <nts:include value="assert.identifier" direction="response" resource="Observation" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4298.2" identifier="99999"/>  
-        <nts:include value="assert.observation.valueq" direction="response" resource="Observation"
-            quantity = "5.4"
-            unit = "µmol/L"
-            system = "http://unitsofmeasure.org"
+        <nts:include value="assert.valuequantity" direction="response" resource="Observation"
+            quantity="5.4"
+            system="http://unitsofmeasure.org"
             code="{µmol/L}"
```

Comment:
The unit 'micromole per liter' is represented by _umol/L_, see https://build.fhir.org/valueset-ucum-units.html. In other parts of the test material also _umol/l_ is used (so it might be good practice to make these consistent).